tools/build_defs/fb_xplat_genrule.bzl

```diff
@@ -1,6 +1,8 @@
 # Only used for PyTorch open source BUCK build
 # @lint-ignore-every BUCKRESTRICTEDSYNTAX
 
+load("@prelude//:rules.bzl", "genrule")
```

Comment:
Rather than do this import, you should do `native.genrule` further down, which should work on both Buck1 and Buck2.